### PR TITLE
cut 2.5.0

### PR DESCRIPTION
**Pull request by anjannath**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 OPENSHIFT_VERSION ?= 4.10.18
 PODMAN_VERSION ?= 4.0.2
 BUNDLE_EXTENSION = crcbundle
-CRC_VERSION = 2.4.1
+CRC_VERSION = 2.5.0
```
Comment (praveenkumar) [inline Makefile:6]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Comment by praveenkumar** [comment]
Wait for CI run

**Comment by cfergeau** [comment]
Are we going to generate new podman bundles ?

**Comment by anjannath** [comment]
> Are we going to generate new podman bundles ?

since the issue with `podman-remote` binary is only on linux, and although i don't like it that it crashes `crc start` but assuming `podman` preset on linux users are very minimal and fedora coreos stream not being available with podman `4.1.1` binaries out of the box, i don't know when we can generate a podman bundle.

i'd prefer to have new podman bundle in the next release

**Comment by praveenkumar** [comment]
> Are we going to generate new podman bundles ?

This is known issue even with last release only 4.1.1 release of podman have static bit correctly available.

**Comment by cfergeau** [comment]
> This is known issue even with last release only 4.1.1 release of podman have static bit correctly available.

Yes, which means we could for example generate a 4.1.0 podman bundle, and use the 4.1.1 linux binary in order to avoid a bad looking bug two releases in a row.

**Pull request merged**